README.md

```diff
@@ -62,34 +68,99 @@ Usage:
-  oomkill           Trace when OOM killer is triggered and kills a process
-  opensnoop         Trace open() system calls
-  process-collector Gather information about running processes
-  profile           Profile CPU usage by sampling stack traces
-  seccomp-advisor   Generate seccomp policies based on recorded syscalls activity
-  sigsnoop          Trace signals received by processes
-  snisnoop          Trace SNI requests
-  socket-collector  Gather information about network sockets
-  tcpconnect        Trace connect() system calls
-  tcptop            Show the TCP traffic in a pod
-  tcptracer         Trace tcp connect, accept and close
-  traceloop         Get strace-like logs of a pod from the past
-  undeploy          Undeploy Inspektor Gadget from cluster
-  version           Show version
+  advise      Recommend system configurations based on collected information
+  audit       Audit a subsystem
+  completion  generate the autocompletion script for the specified shell
+  deploy      Deploy Inspektor Gadget on the cluster
+  help        Help about any command
+  profile     Profile different subsystems
+  snapshot    Take a snapshot of a subsystem and print it
+  top         Gather, sort and print events according to a given criteria
+  trace       Trace and print system events
+  traceloop   Get strace-like logs of a pod from the past
+  undeploy    Undeploy Inspektor Gadget from cluster
+  version     Show version
+
+...
+```
+
+You can then get help for each subcommand:
+
+```bash
+$ kubectl gadget advise --help
+Recommend system configurations based on collected information
+
+Usage:
+  kubectl-gadget advise [command]
+
+Available Commands:
+  network-policy  Generate network policies based on recorded network activity
+  seccomp-profile Generate seccomp profiles based on recorded syscalls activity
+
+...
+$ kubectl gadget audit --help
+Audit a subsystem
+
+Usage:
+  kubectl-gadget audit [command]
+
+Available Commands:
+  seccomp     Trace syscalls that seccomp sent to the audit log
+
+...
+$ kubectl gadget profile --help
+Profile different subsystems
+
+Usage:
+  kubectl-gadget profile [command]
+
+Available Commands:
+  block-io    Generate a histogram with the distribution of block device I/O latency
+  cpu         Profile CPU usage by sampling stack traces
+
+...
+$ kubectl gadget snapshot --help
+Take a snapshot of a subsystem and print it
+
+Usage:
+  kubectl-gadget snapshot [command]
+
+Available Commands:
+  process     Gather information about running processes
+  socket      Gather information about network sockets
+
+...
+$ kubectl gadget top --help
+Gather, sort and print events according to a given criteria
+
+Usage:
+  kubectl-gadget top [command]
+
+Available Commands:
+  block-io    Trace block devices I/O, with container details
+  file        Trace reads and writes by file, with container details
+  tcp         Trace TCP connection, with container details
```

Comment:
No related to this PR: we should remove `with container details`. All our gadgets provide this information, then it's not needed.